lms/static/js/edxnotes/views/shim.js

```diff
@@ -58,6 +58,31 @@ define([
         return (timeToExpiry > 0) ? timeToExpiry : 0;
     };
 
+    Annotator.Plugin.Tags.prototype.pluginInit = _.compose(
+        function() {
+            // Add screen reader label for edit mode.
+            var index = this.annotator.editor.fields.length -1;
+            var tagsField = 'annotator-field-' + index;
+            $('<label class="sr" for='+ tagsField +'>Edit tags</label>').insertBefore(
```

Comment:
@cahrens, yes, we have been trying to internationalize screen reader text as much as possible (sometimes we forget).